.github/workflows/ci.yml

```diff
@@ -7,9 +7,29 @@ on:
     branches:
       - master
 jobs:
+
+  pre-build:
+    if: github.actor == 'dependabot[bot]'
+    name: Check expo dependencies
+    uses: ./.github/workflows/check-expo-dependencies.yml
+
   build:
     runs-on: ubuntu-latest
+    needs: pre-build
+    if: always()
```

Comment:
I don't think this if is needed

Reply:
As the `build` job depends on the `pre-build` if the last one is not executed (if it is not a PR from dependabot, for example) it was not being executed either. So the `if: always()` ensures that the `build` job runs regardless of whether the `pre-build` happens or not.